app/modules/entities/components/layouts/entity_title.svelte

```diff
@@ -13,7 +14,7 @@
 </script>
 <h2>
   {#if standalone}
-    {label}
+    {capitalize(label)}
```

Comment:
Entities labels should already have their case set, capitalizing every label can lead to undesired capitalizations

Reply:
done in b1594019e